main.py

```diff
@@ -88,6 +88,7 @@ def experiment(
         logger=logger,
         max_epochs=epochs,
         precision=16 if use_amp else 32,
+        log_every_n_steps=1,
```

Comment:
by default pytorch lightning logs every 50 iterations. So even if they're doing everything right, they may not see it